### DISCUSSION TITLE
Requesting feedback on having an admin portal

### DISCUSSION THREAD

**Discussion by kurtaking** [procore-oss/backstage-plugin-announcements#234] [Ideas]
Hey, I started working on the concept of a single location to manage announcements in https://github.com/procore-oss/backstage-plugin-announcements/pull/130/. 

I'm posting this looking to gauge the level of interest for a feature like this.

**Comment by billabongrob** [comment]
This looks pretty cool.  I think it also lends itself to other functionality, such as a simple approval system.

**Reply by kurtaking** [reply]
👋🏼 Glad you took some interest in this. I haven't looked at this since earlier this year and will have limited capacity to contribute new functionality until the end of June because I'll be learning to be a dad. 😄 I still think it's a good idea and centralizes things if someone wants to pick it up.

**Reply by billabongrob** [reply]
Congrats! 🎉 👏  It aint easy but its worth it! I'll try to help when possible.
Some other thoughts would be being able to adjust the owner of an announcement (this has happened to us recently, which is an easy DB update.

**Comment by kurtaking** [comment]
The latest release will include a version of this for people to start using. [PR](https://github.com/procore-oss/backstage-plugin-announcements/pull/461)
